rigex-front-end/src/store/signUp/actionTypes.js

```diff
@@ -1,5 +1,5 @@
 const actionTypes = {
-  SET_USER: 'SET_LOGGED_IN_USER',
+  SET_NEW_USER: 'SET_SIGNED_UP_USER',
```

Comment:
this is overwriting actions I need in store/login/actionTypes for login functionality. Supposed to be in store/register/actionTypes?

Reply:
Oh I see, the filename changed? I might be reading the github logs wrong